prov/sockets/src/sock_ep_msg.c

```diff
@@ -432,14 +432,13 @@ static int sock_ep_cm_send_msg(struct sock_cm_entry *cm,
 			       void *msg, size_t len)
 {
 	int ret;
+#if ENABLE_DEBUG
 	char sa_ip[INET_ADDRSTRLEN] = {0};
-
 	memcpy(sa_ip, inet_ntoa(addr->sin_addr), INET_ADDRSTRLEN);
-	SOCK_LOG_DBG("Sending message to %s:%d\n", sa_ip, ntohs(addr->sin_port));
-
+#endif
 	ret = sendto(cm->sock, (char *) msg, len, 0,
 		     (struct sockaddr *) addr, sizeof(*addr));
-	SOCK_LOG_DBG("Total Sent: %d\n", ret);
+	SOCK_LOG_DBG("Sending %d to %s:%d\n", ret, sa_ip, ntohs(addr->sin_port));
```

Comment:
The log has to be after the <code>sendto</code> call to get the <code>ret</code> value. I can move the #ifdef down but then <code>char sa_ip[INET_ADDRSTRLEN] = {0}</code> declaration needs to be moved down. Let me know if that's alright.